### PR TITLE
Add missing include

### PR DESCRIPTION
**Pull request by JoelLinn**
Needed for std::unique_ptr. Building was broken previously on Ubuntu 18.04.

### CODE REVIEW

**Comment by gvollant** [comment]
@kevwongg-goog can pehaps take a loot, @penmetsaa

**Comment by gvollant** [comment]
@JoelLinn I suggest you rebase against master

**Comment by gvollant** [comment]
@JoelLinn  again,  I suggest you rebase against master
I suggest, if possible, @kevwongg-goog take a look and review, @penmetsaa 

for info, at https://en.cppreference.com/w/cpp/memory/unique_ptr we can check std::unique_ptr need memory header

**Review by gvollant — APPROVED** [review]
ok for me

**Review by lararennie — APPROVED** [review]

**Comment by JoelLinn** [comment]
Rebased.

**Review by gvollant — DISMISSED** [review]

**Comment by gvollant** [comment]
@lararennie can you approve again after the rebase? thank!

**Review by lararennie — DISMISSED** [review]

**Comment by JoelLinn** [comment]
I am sorry but why is it so hard to merge a missing include? :/

**Comment by gvollant** [comment]
@penmetsaa can do it I beleive

**Comment by lararennie** [comment]
I think it needs rebasing. After that: if it doesn't let you merge I can click that too, I expected you to be able to do that.

**Comment by penmetsaa** [comment]
Thanks so much for the contribution. Sorry for the delay here; because of some setup issues at my end,
I could not test / look into it. To double check:
Followed regular CPP build process with this change and all tests passed.

**Review by penmetsaa — APPROVED** [review]

**Comment by penmetsaa** [comment]
Please make the branch sync with master. Will merge after it.

**Comment by gvollant** [comment]
I just added a very very trivial warning fix #2585 (unused parameter)

https://github.com/google/libphonenumber/pull/2585

**Review by gvollant — APPROVED** [review]

**Review by lararennie — APPROVED** [review]

**Pull request merged**
